store/job_storer_test.go

```diff
@@ -0,0 +1,58 @@
+package store
+
+import (
+	"context"
+	"github.com/ONSdigital/dp-search-reindex-api/models"
+	"testing"
+	"time"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+// Constants for testing
+const (
+	testJobID1 = "UUID1"
+)
+
+var ctx = context.Background()
+
+func TestCreateJob(t *testing.T) {
+	Convey("Successfully return without any errors", t, func() {
+		Convey("when the job has the id field for POST request", func() {
+			inputID := testJobID1
+
+			jobStorer := JobStorer{}
+			job, err := jobStorer.CreateJob(ctx, inputID)
+			So(err, ShouldBeNil)
+			expectedJob := expectedJob()
+			So(job.ID, ShouldEqual, expectedJob.ID)
+			So(job.Links, ShouldResemble, expectedJob.Links)
+			So(job.NumberOfTasks, ShouldEqual, expectedJob.NumberOfTasks)
+			So(job.ReindexCompleted, ShouldEqual, expectedJob.ReindexCompleted)
+			So(job.ReindexFailed, ShouldEqual, expectedJob.ReindexFailed)
+			So(job.ReindexStarted, ShouldEqual, expectedJob.ReindexStarted)
+			So(job.SearchIndexName, ShouldEqual, expectedJob.SearchIndexName)
+			So(job.State, ShouldEqual, expectedJob.State)
+			So(job.TotalSearchDocuments, ShouldEqual, expectedJob.TotalSearchDocuments)
+			So(job.TotalInsertedSearchDocuments, ShouldEqual, expectedJob.TotalInsertedSearchDocuments)
+		})
+	})
+}
+
+func expectedJob() models.Job {
```

Comment:
3rd time you've had to create a job with default values like this - consider making a `NewJob(id string)` func to do this for you.

Reply:
done - I've now removed the expectedJob function altogether as the NewJob can be used instead